src/components/Select/types.ts

```diff
@@ -1,6 +1,12 @@
-import {TextInputSize, TextInputView} from '../TextInput';
+import React from 'react';
+import {TextInputView, TextInputSize, TextInputPin} from '../TextInput';
 
 export type SelectProps = {
     view?: TextInputView;
     size?: TextInputSize;
+    pin?: TextInputPin;
+    controlWidth?: 'unset' | 'auto' | 'max' | number;
```

Comment:
`'unset'` is not the same as `undefined` here?

Reply:
It is not the same because we want have `controlWidth : 'auto'` by default. This means that `undefined` value automatically converted to `'auto'`